ivy/functional/frontends/paddle/tensor/search.py

```diff
@@ -85,3 +85,20 @@ def masked_select(x, mask, name=None):
 )
 def topk(x, k, axis=None, largest=True, sorted=True, name=None):
     return ivy.top_k(x, k, axis=axis, largest=largest, sorted=sorted)
+
+
+# where
+@to_ivy_arrays_and_back
+def where(condition, x=None, y=None, name=None):
+    if x is not None and y is not None:
+        return ivy.where(condition, x, y)
+    else:
+        raise ValueError("Both x and y should be given.")
```

Comment:
```suggestion
@to_ivy_arrays_and_back
def where(condition, x, y, name=None):
    ivy.where(condition, x, y)
```
It's better to just make the arguments required and let python check them rather than doing it yourself in the function